static/src/stylesheets/module/identity/_ad-prefs.scss

```diff
@@ -64,7 +60,7 @@
 }
 
 .identity-ad-prefs-manager__footer {
-    margin-top: $gs-gutter * 2;
+    margin-top: $gs-gutter;
```

Comment:
this should be a multiplier of `gs-baseline`  as `gutter` is for horizontal margins. Sorry about that! i wrote the original wrongly